db/structure.sql

```diff
@@ -416,7 +416,7 @@ SET default_tablespace = '';
 SET default_with_oids = false;
 
 --
--- Name: auctions; Type: TABLE; Schema: audit; Owner: -; Tablespace: 
```

Comment:
Should I remove this from PR and leave only migration stamp?

Reply:
Definitely, if this change is not connected to this PR.